internal/postgres/hba.go

```diff
@@ -25,7 +25,7 @@ func NewHBAs() HBAs {
 	return HBAs{
 		Mandatory: []HostBasedAuthentication{
 			// The "postgres" superuser must always be able to connect locally.
-			*NewHBA().Local().User("postgres").Method("peer"),
+			*NewHBA().Local().User("postgres").Method("trust"),
```

Comment:
why this change?

Reply:
because we can't connect via socket from pmm-client. Anyway I think it is not good a approach to change it and we need to improve it in the next release
```
peer
Obtain the client's operating system user name from the operating system and check if it matches the requested database user name. This is only available for local connections. See [Section 21.9](https://www.postgresql.org/docs/current/auth-peer.html) for details.
```